components/HelpModal.go

```diff
@@ -0,0 +1,77 @@
+package components
+
+import (
+	"github.com/gdamore/tcell/v2"
+	app "github.com/jorgerojas26/lazysql/app"
+	"github.com/jorgerojas26/lazysql/commands"
+	. "github.com/jorgerojas26/lazysql/keymap"
+	"github.com/rivo/tview"
+)
+
+type HelpModal struct {
+	*tview.Flex
+}
+
+func NewHelpModal() *HelpModal {
+
+	//Colors
```

Comment:
Go comments are starting by a space.
Otherwise, they are supposed to be compiler or language instructions


```suggestion
	// Colors
```

revive can help in reporting such issues